online_store/gt_sport_v2.js

```diff
@@ -87,21 +92,22 @@ function productDescriptionDisplay() {
       <p> <button type="button" id="addToShoppingCartBtn">Add to Shopping Cart</button>
```

Comment:
**Recommendation**
Here you should generate different id for each button because id are always unique.
for the problem that you are facing have 2 solution.
_Solution 1:_
generate different id for each button like: addToShoppingCartBtn0, addToShoppingCartBtn1, etc.
then after you render your component you can addEventListener **click** to each button.
_Solution 2:_
you can read a bit about [onclick](https://www.w3schools.com/tags/ev_onclick.asp) event and use it in your button, this way you don't need to add ids.

Reply:
Thanks, I will go with 'onclick' event.